### DISCUSSION TITLE
What file formats does the modrinth suport sharing for modpacks?

### DISCUSSION THREAD

**Discussion by Mr-Jokes-3** [modrinth/.github#177] [Other discussions]
What file formats does the modrinth suport sharing for modpacks, every time I try to upload a modpack I made with curseforge launcher, it does not work, I keep getting this notification
![image](https://github.com/modrinth/.github/assets/143090121/c09c7513-1fc9-430c-ad6c-f19d8b16f197)
(I understand that i canot launch it via the modrinth launcher BTW)

**Comment by triphora** [comment]
https://docs.modrinth.com/modpacks

**Reply by Mr-Jokes-3** [reply]
thanks
